Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -138,10 +138,9 @@ - (void)genericHTTPRequest:(NSURLRequest *)request retryNumber:(NSInteger)retryN
             }
             
         }
-        dispatch_async(dispatch_get_main_queue(), ^{
-            if (callback)
-                callback(serverResponse, error);
-        });
+		//	Don't call on the main queue since it might be blocked.
+		if (callback)
```

Comment:
Ah.  I must have an editor that is converting spaces to tabs or vice versa.  I can fix that.